docling_core/types/doc/document.py

```diff
@@ -461,12 +535,7 @@ def export_to_markdown(
                     continue
 
                 item_type = item.obj_type
-                if isinstance(item, BaseText) and item_type in {
-                    "title",
-                    "subtitle-level-1",
-                    "paragraph",
-                    "caption",
-                }:
+                if isinstance(item, BaseText) and item_type in {}:
```

Comment:
I guess this was intended this way:

```suggestion
                if isinstance(item, BaseText) and item_type in main_text_labels:
```